pkg/util/containers/filter.go

```diff
@@ -14,26 +14,39 @@ import (
 )
 
 const (
-	// pauseContainerGCR regex matches:
+	// Pause container image names that should be filtered out.
+	// Where appropriate, each constant is loosely structured as
+	// image:domain(.*)/pause(.*)
+
+	pauseContainerKubernetes = "image:kubernetes/pause"
+	pauseContainerECS        = "image:amazon/amazon-ecs-pause"
+	pauseContainerOpenshift  = "image:openshift/origin-pod"
+	pauseContainerOpenshift3 = "image:(.*)rhel7/pod-infrastructure"
+
 	// - k8s.gcr.io/pause-amd64:3.1
 	// - asia.gcr.io/google_containers/pause-amd64:3.0
 	// - gcr.io/google_containers/pause-amd64:3.0
 	// - gcr.io/gke-release/pause-win:1.1.0
-	pauseContainerGCR        = `image:(.*)gcr\.io(/google_containers/|/gke-release/|/)pause(.*)`
-	pauseContainerOpenshift3 = "image:(openshift/origin-pod|(.*)rhel7/pod-infrastructure)"
-	pauseContainerKubernetes = "image:kubernetes/pause"
-	pauseContainerECS        = "image:amazon/amazon-ecs-pause"
-	pauseContainerEKS        = "image:(amazonaws.com/)?eks/pause-(amd64|windows)"
-	// pauseContainerAzure regex matches:
+	// - eu.gcr.io/k8s-artifacts-prod/pause:3.3
+	pauseContainerGCR = `image:(.*)gcr\.io(.*)/pause(.*)`
+
 	// - k8s-gcrio.azureedge.net/pause-amd64
 	// - gcrio.azureedge.net/google_containers/pause-amd64
-	pauseContainerAzure   = `image:(.*)azureedge\.net(/google_containers/|/)pause(.*)`
+	pauseContainerAzure = `image:(.*)azureedge\.net(.*)/pause(.*)`
+
+	// amazonaws.com/eks/pause-windows:latest
+	// eks/pause-amd64
+	pauseContainerEKS = `image:(amazonaws\.com/)?eks/pause-(amd64|windows)`
```

Comment:
We should try to spawn an [EC2 A1 instance](https://aws.amazon.com/ec2/instance-types/a1) inside an Amazon EKS cluster, to check if we would get an `eks/pause-arm64` image ?

```suggestion
	pauseContainerEKS = `image:(amazonaws\.com/)?eks/pause-(amd64|arm64|windows)`
```

Reply:
good point - found evidence that it exists here https://github.com/aws/containers-roadmap/issues/264